src/cli/cmd/add/flake.rs

```diff
@@ -275,6 +275,15 @@ impl AttrType {
         flake_contents: &str,
     ) -> color_eyre::Result<String> {
         let mut new_flake_contents = flake_contents.to_string();
+
+        if head
+            .arguments
+            .iter()
+            .any(|arg| &*arg.identifier == flake_input_name)
+        {
```

Comment:
I'd normally add a `tracing::trace!("Found existing duplicate input")` here so when I got very confused later why my code was exiting early I might actually notice.

Reply:
We'd have to setup tracing first...! I'll make a ticket for that.